contracts/solidity/test/mock_contracts/mock_SN_messaging.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.21;
+
+contract mock_IStarknetMessaging {
+    function sendMessageToL2(
+        uint256 toAddress,
+        uint256 selector,
+        uint256[] calldata payload
+    ) external payable returns (bytes32, uint256) {
+        require(msg.value > 0, "L1_MSG_FEE_MUST_BE_GREATER_THAN_0");
+        require(msg.value <= 1 ether, "MAX_L1_MSG_FEE_EXCEEDED");
```

Comment:
Why do we have this restriction here?

Reply:
This restrictions are the same as the SN Core Contract.
I was considering changing this approach (mocking the core contract and calling it) to what is done here (hardcoding a mocked answer of the core contract):

https://github.com/yetanotherco/yet-another-bridge/pull/102/files#diff-4cb54a4b5b8cdf00df6e5c8a3cb7739d9dadf5550b1288d064d7c35a027ee722R20-R24

What do you think?

Reply:
I agree it's a better approach. Good idea